src/webextension/list/popup/components/app.css

```diff
@@ -5,14 +5,16 @@
 body {
   padding: 0;
   margin: 0;
+  width: 280px;
+  height: 360px;
   color: #0c0c0d;
   background-color: #ededf0;
   font: caption;
   font-size: 13px;
   -moz-user-select: none;
 }
 
+body > main,
 .app {
-  width: 280px;
-  height: 400px;
+	height: 100%;
```

Comment:
maybe I'm crazy but should this backdent one? if so: no need to fix now, but if someone notices it in another round of revisions.

Reply:
Looks like a tab character snuck in.

Reply:
Doh...my tab size was set back at 4 somehow...my bad

Reply:
No problemo. 1fe3cc924ef083ec1fd85543d56699002a2ca97a